### PR TITLE
Build and runtime fixes

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: pkg/image/image.go [hunk 1]
```diff
@@ -59,16 +62,22 @@ func NewUDF(cli cli.Commander) *UDF {
 
 // Create makes the required call to UDF to
 func (u *UDF) Create(release, channel, arch string, ver int) (path string, err error) {
-	tmpDirName, _ := u.cli.ExecCommand("mktemp -d")
-	tmpFileName := filepath.Join(tmpDirName, outputFileName)
+	tmpDirName, err := u.cli.ExecCommand("mktemp", "-d")
+	if err != nil {
+		return
+	}
+	tmpFileName := filepath.Join(strings.TrimSpace(tmpDirName), outputFileName)
```
Comment (come-maiz) [inline pkg/image/image.go:69]: is it possible for mktemp to return spaces? Not sure why are you trimming here.

**Comment by come-maiz** [comment]
looks good. Just a question in  there.

File: pkg/image/image.go (hunk 1 above)
Comment (fgimenez) [inline pkg/image/image.go:69]: It was returning a \n at the end, maybe trimspace is too strong for just that :) 
Thanks!

Comment (come-maiz) [inline pkg/image/image.go:69]: heh, maybe yes :) You could make it clearer with TrimSuffix.

**Pull request merged**